common/lib/xmodule/xmodule/modulestore/courseware_index.py

```diff
@@ -129,7 +129,7 @@ def remove_index_item_location(item_location):
             error_list.append(_('General indexing error occurred'))
 
         if raise_on_error and error_list:
-            raise SearchIndexingError(_('Error(s) present during indexing'), error_list)
+            raise SearchIndexingError(_('Errors present during indexing'), error_list)
```

Comment:
Because `(s)` will cause Transifex to force the translated text to have `()` too. It is not quite necessary to keep the `(s)` here.